### PR TITLE
fix: sourcepawn grammar update

### PR DESCRIPTION
**Pull request by Sarrus1**
Fixes the issue raised in https://github.com/nvim-treesitter/nvim-treesitter/pull/6246.

### CODE REVIEW

**Comment by clason** [comment]
Thanks! You need to update the hash in the lockfile, too.

File: lockfile.json [hunk 1]
```diff
@@ -797,4 +797,4 @@
   "zig": {
     "revision": "0d08703e4c3f426ec61695d7617415fff97029bd"
   }
-}
+}
```
Comment (lucario387) [inline lockfile.json:800]: Hehh

**Review by clason — APPROVED** [review]

File: lockfile.json (hunk 1 above)
Comment (clason) [reply lockfile.json:800]: We'll fix it in post.

Comment (Sarrus1) [reply lockfile.json]: Autoformat was on :/

**Pull request merged**